src/test/java/io/vertx/rabbitmq/RabbitMQClientTestBase.java

```diff
@@ -26,11 +28,12 @@
   protected RabbitMQClient client;
   protected Channel channel;
   protected Vertx vertx;
+  protected RabbitMQManagementClient managementClient;
 
   @ClassRule
-  public static final GenericContainer rabbitmq = new FixedHostPortGenericContainer<>("rabbitmq:3.7")
+  public static final GenericContainer rabbitmq = new FixedHostPortGenericContainer<>("rabbitmq:3.7-management")
```

Comment:
why is this change needed  ? more curious than questioning it

Reply:
I use management api to test bind and unbind methods.

Reply:
Any reason to used a Fixed Port container? I know it was there before, but maybe a good time to change this.

Reply:
No, there is no reason to use fixed port container. I'll make changes to use random port.

Reply:
@cescoffier Fixed ports here are exposed ports. Mapped ports are already random.